contrib/check-null-false-returns.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/python3
```

Comment:
shouldn't this be in `contrib/ci` if it's for CI purposes.

Reply:
Yes! Fixed!